tests/common.py

```diff
@@ -310,13 +310,13 @@ async def _await_count_and_log_pending(
 
     # Load the registries
     if load_registries:
```

Comment:
I wonder if this ever is used in the false state.

Reply:
Yes, it is :) Quite bit actually